src/vos/vos_pool.c

```diff
@@ -63,6 +63,19 @@ vos_pool_settings_init(void)
 	return umempobj_settings_init();
 }
 
+static inline enum bio_mc_flags
+vos2mc_flags(unsigned int vos_flags)
+{
+	enum bio_mc_flags mc_flags = 0;
+
+	if (vos_flags & VOS_POF_SYSDB)
+		mc_flags |= BIO_MC_FL_SYSDB;
+	if (vos_flags & VOS_POF_RDB)
+		mc_flags |= BIO_MC_FL_RDB;
+
+	return mc_flags;
+}
+
 static int
 vos_pmemobj_create(const char *path, uuid_t pool_id, const char *layout,
```

Comment:
maybe could add path_size as an arg, so we don't need call stat twice.

Reply:
right, let's cleanup it in follow-on patches.